Add page for DMR (#17)
SolaceDev/maas-docs-online: docs/group_concepts/gc_dmr.rst

Dynamic Message Routing in PubSub+ Cloud
=======================================

In PubSub+ Cloud, all enterprise messaging services are created with the Dynamic Message Routing (DMR) feature enabled. To learn more about DMR,

> checkout `the official documentation in the Solace Tech Docs <https://docs.solace.com/Features/Dynamic-Msg-Routing.htm>`_.

Reviewer: Should be check out.

> PubSub+ Cloud will only prepare services for DMR in the following plans:

Reviewer: ```suggestion
The following PubSub+ Cloud plans are DMR enabled:
```

- Kilo
- Mega
- Giga
- Tera

> The message VPN created by PubSub+ Cloud has DMR enabled. A DMR cluster is automatically configured in the message VPN for the service. The password

Reviewer: ```suggestion
The messaging service created by PubSub+ Cloud is DMR enabled. A DMR cluster is automatically configured on the PubSub+ Broker and enabled within the message VPN. The password
```

> for cluster communication is stored in PubSub+ Cloud and available from the status tab on the service details page, along with all other relevant DMR cluster

Reviewer: ```suggestion
for cluster communication is stored in PubSub+ Cloud and available from the **Status** tab on the service details page, along with all other relevant DMR cluster
```

information.

> Note that in PubSub+ Cloud, only external links can be created for DMR clusters at this time. To create DMR cluster links, use the 

Reviewer: ```suggestion
Note that in PubSub+ Cloud, you should only use external links to create event meshes. Internal links will be managed for you by PubSub+ Cloud. To create DMR cluster links, use the 
```
